esp/services/ws_ecl/ws_ecl_service.hpp

```diff
@@ -87,11 +87,21 @@ typedef enum wsEclTypes_
 
 } wsEclType;
 
+class RoxieConnEndpoint : public CSmartSocketFactory
```

Comment:
minor: These class names don't seem very related, yet they are publicly inheriting.  It makes me wonder if this information is in the correct place.